package.json

```diff
@@ -4,25 +4,27 @@
   "description": "treemap visualization",
   "main": "dist/webtreemap.js",
   "bin": {
-    "webtreemap": "build/cli.js"
+    "webtreemap": "build-node/cli.js"
   },
   "dependencies": {
-    "commander": "^2.11.0"
+    "commander": "^2.11.0",
+    "open": "^8.2.1"
   },
   "devDependencies": {
     "@types/commander": "^2.11.0",
-    "@types/node": "^8.0.34",
+    "@types/node": "12.20.20",
```

Comment:
This update includes the `promises` bit of the `fs` module. (I believe this was added in Node 10.)